osx/cookbooks/vim/default.rb

```diff
@@ -0,0 +1,7 @@
+package 'vim' do
+  options '--with-lua --with-luajit'
+end
+
+git '~/.vim' do
+  repository 'https://github.com/tacahilo/.vim'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.